crates/summit/src/routes.rs

```diff
@@ -31,6 +31,8 @@ pub struct TasksQuery {
     pub page: Option<NonZeroU32>,
     pub per_page: Option<u32>,
     pub status: Option<task::Status>,
+    pub sort: Option<String>,
+    pub order: Option<String>,
```

Comment:
We should strongly type these within `task::query` module and use them with that API for doing the database sorting